src/sqerl_core.erl

```diff
@@ -0,0 +1,260 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Mark Paradise <marc.paradise@chef.io>
+%% @author Seth Falcon <seth@chef.io>
+%% @author Marc Anderson <mark@chef.io>
+%% Copyright 2011-2015 Chef Software, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+-module(sqerl_core).
```

Comment:
Since we are changing arity of the functions, pros/cons of breaking out the separate module?